articles/architecture-scenarios/_includes/_logout/_introduction.md

```diff
@@ -33,3 +33,7 @@ If the user logs out and you redirect them back to the application, and the appl
 ## Automatic termination of sessions
 
 Not all users will trigger the logout process manually, so Auth0 also provides **session timeout** to prevent overly long-lived sessions. This setting is [available and configurable via the Auth0 Dashboard](/dashboard/dashboard-tenant-settings#session-timeout).
+
+<% if (platform === "b2b") { %>
+If you are redirecting users to a separate IDP to login then [Federated User Logout](/logout/guides/logout-idps) is something that you need to consider for your application.  *Should* you log the user out of their IDP when they log out of your application?  The answer to this depends on what your users would expect. If the application is tied closely to the organization and is a central part, then they may expect to be logged out of their IDP, if not, then it may be frustrating to them to get logged out of their IDP when they log out of your application. In most B2B applications, federated logout is not expected by the user.
```

Comment:
```suggestion
[Federated User Logout](/logout/guides/logout-idps) may be something something that you need to consider for your application. If you or your customers will be using a 3rd party Identity Provider (IdP) - i.e. something other than a [Database Identity Provider](/connections/database) in Auth0 - then the question of wether you need to log the user out of the IdP when they log out of your application is something you will need to answer. The answer depends on what your users would expect. If the application and/or IdP used is tied closely to a customer organization and a central part of day-to-day operations, then it may be frustrating for users to get logged out of their IDP when they log out of your application. If not, then being logged out of the IDP may be expected - or in some cases even desired. In most B2B scenarios, our customers find that it is preferable *not* to perform federated logout for a user.
```

Reply:
Added to separate PR